src/js/components/Button/Button.js

```diff
@@ -70,13 +71,20 @@ class Button extends Component {
           ],
       });
     }
+
+    let domTag = tag;
```

Comment:
can we replace it with a ternary "if" ?

`const domTag = !tag && href ? 'a' : undefined;`